client/src/Loader/Loader.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./Loader.css";
+import { Transition } from "react-transition-group";
+const Loader = ({ visible }) => {
+    return (
+        <div className={visible ? "app visible" : "app"}>
```

Comment:
ternary not needed

Reply:
fixed

Reply:
https://let-is-deploy-lab-2.herokuapp.com/